src/repository/exercisesRepository.ts

```diff
@@ -325,6 +325,426 @@ export const db: Schema = {
       Nós: 'jogamos',
       'Eles/Elas/Vocēs': 'jogam',
       exerciseLevel: 1
+    },
+    {
+      english: 'to write',
+      infinitive: 'escrever',
+      Eu: 'escrevo',
+      Tu: 'escreves',
+      'Ela/Ele/Você': 'escreve',
+      Nós: 'escrevemos',
+      'Eles/Elas/Vocēs': 'escrevem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to understand',
+      infinitive: 'compreender',
+      Eu: 'compreendo',
+      Tu: 'compreendes',
+      'Ela/Ele/Você': 'compreende',
+      Nós: 'compreendemos',
+      'Eles/Elas/Vocēs': 'compreendem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to know',
+      infinitive: 'conhecer',
+      Eu: 'conheço',
+      Tu: 'conheces',
+      'Ela/Ele/Você': 'conhece',
+      Nós: 'conhecemos',
+      'Eles/Elas/Vocēs': 'conhecem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to solve',
+      infinitive: 'resolver',
+      Eu: 'resolvo',
+      Tu: 'resolves',
+      'Ela/Ele/Você': 'resolve',
+      Nós: 'resolvemos',
+      'Eles/Elas/Vocēs': 'resolvem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to go down',
+      infinitive: 'descer',
+      Eu: 'desço',
+      Tu: 'desces',
+      'Ela/Ele/Você': 'desce',
+      Nós: 'descemos',
+      'Eles/Elas/Vocēs': 'descem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to warm up',
+      infinitive: 'aquecer',
+      Eu: 'aqueço',
+      Tu: 'aqueces',
+      'Ela/Ele/Você': 'aquece',
+      Nós: 'aquecemos',
+      'Eles/Elas/Vocēs': 'aquecem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to live',
+      infinitive: 'viver',
+      Eu: 'vivo',
+      Tu: 'vives',
+      'Ela/Ele/Você': 'vive',
+      Nós: 'vivemos',
+      'Eles/Elas/Vocēs': 'vivem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to run',
+      infinitive: 'correr',
+      Eu: 'corro',
+      Tu: 'corres',
+      'Ela/Ele/Você': 'corre',
+      Nós: 'corremos',
+      'Eles/Elas/Vocēs': 'correm',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to learn',
+      infinitive: 'aprender',
+      Eu: 'aprendo',
+      Tu: 'aprendes',
+      'Ela/Ele/Você': 'aprende',
+      Nós: 'aprendemos',
+      'Eles/Elas/Vocēs': 'aprendem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to forget',
+      infinitive: 'esquecer',
+      Eu: 'esqueço',
+      Tu: 'esqueces',
+      'Ela/Ele/Você': 'esquece',
+      Nós: 'esquecemos',
+      'Eles/Elas/Vocēs': 'esquecem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to use',
+      infinitive: 'usar',
+      Eu: 'uso',
+      Tu: 'usas',
+      'Ela/Ele/Você': 'usa',
+      Nós: 'usamos',
+      'Eles/Elas/Vocēs': 'usam',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to have lunch',
```

Comment:
suggestion: you can also add "to have dinner" = "jantar"

Reply:
added `jantar` :+1: